### DISCUSSION TITLE
Trivy operator schedule scan

### DISCUSSION THREAD

**Discussion by chary1112004** [aquasecurity/trivy-operator#1743] [Help & Support]
Hi,

Currently we detect trivy operator transfer data through NAT gateway (we are using AWS) quite amount data then it really cost.
Then we would like trivy operator just schedule scan like 1 time a day, not scanning whenever new pod is deployed. Does trivy operator support this case?

Helm chart installed: https://artifacthub.io/packages/helm/trivy-operator/trivy-operator/0.3.0

Any suggestion to resolve this issue would be very much appreciated!

Thanks

**Comment by chen-keinan** [comment]
@chary1112004 see my comment on issue , trivy-operator v0.18.0 use cache
